components/StatusDropdown/styles.ts

```diff
@@ -0,0 +1,77 @@
+import styled from 'styled-components';
+import COLORS from '../../styles/colors';
+
+export const FilterDropdownStyles = styled.div`
+  position: relative;
+  background: ${COLORS.white};
+  width: 10.5rem;
+  border-radius: 0.5rem;
+  padding-bottom: 0.5rem;
+`;
+
+export const OptionTitleStyles = styled.p<{ color: string }>`
+  background: ${({ color }) => `${color}10`};
+  border-radius: 0.3rem;
+  padding: 0.1rem 0.4rem;
+  display: inline-block;
+`;
+
+export const CheckboxStyles = styled.input`
+  cursor: pointer;
+`;
+
+export const ApplyButtonStyles = styled.button<{ isActive: boolean }>`
+  width: 8rem;
+  height: 1.5rem;
+  flex-shrink: 0;
+  border-radius: 0.25rem;
+  background: ${({ isActive }) =>
+    isActive
+      ? COLORS.electricBlue
+      : 'rgba(73, 116, 224, 0.50)'}; // Active vs Inactive color
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  cursor: ${({ isActive }) => (isActive ? 'pointer' : 'not-allowed')};
+  border: none;
+  margin: 1.4rem auto 0.5rem auto;
```

Comment:
auto not 0.5rem (we don't need this margin since we have a margin for the div wrapped around all the content

Reply:
It should look like `margin: 1.4rem auto auto auto;`